traffic_ops/testing/api/v14/user_test.go

```diff
@@ -42,12 +45,59 @@ func CreateTestUsers(t *testing.T) {
 
 		resp, _, err := TOSession.CreateUser(&user)
 		if err != nil {
-			t.Fatalf("could not CREATE user: %v\n", err)
+			t.Errorf("could not CREATE user: %v\n", err)
 		}
 		log.Debugln("Response: ", resp.Alerts)
 	}
 }
 
+func RolenameCapitalizationTest(t *testing.T) {
+
+	var err error
+	var jsonBytes []byte
+
+	user := tc.User{}
+	blob := []byte(`
+	{
+		"username": "test_user",
+		"email": "cooldude6@comcast.net",
+		"fullName": "full name is required",
+		"localPasswd": "better_twelve",
+		"confirmLocalPasswd": "better_twelve",
+		"role": 1,
+		"tenantId": 1
+	}`)
+	err = json.Unmarshal(blob, &user)
+	if err != nil {
+		t.Errorf("could not unmarshal json into struct: %v\n", err)
+	}
+
+	resp, _, err := TOSession.CreateUser(&user)
+	if err != nil {
+		t.Errorf("could not CREATE user: %v\n", err)
+	}
+	jsonBytes, err = json.Marshal(resp)
+	if err != nil {
+		t.Errorf("coult not marshal struct to bytes: %v\n", err)
+	}
+	if bytes.Contains(jsonBytes, []byte("rolename")) {
+		t.Errorf("incorrect json was returned for PUT")
+	}
+
+	resp2, _, err := TOSession.GetUserByUsername(*user.Username)
+	if err != nil {
+		t.Errorf("could not GET user: %v\n", err)
+	}
+	jsonBytes, err = json.Marshal(resp2)
+	if err != nil {
+		t.Errorf("coult not marshal struct to bytes: %v\n", err)
+	}
+	if bytes.Contains(jsonBytes, []byte("roleName")) {
```

Comment:
ditto the comment above, would probably be better to fail the test if the response does NOT contain `"rolename": "<insert role name here>"`.